### PR TITLE
Create Triangles.m

### PR DESCRIPTION
**Pull request by mitemijalkov**
Completed the class Triangles.

### CODE REVIEW

File: braph/graph/measures/Triangles.m [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    methods
+        function m = Triangles(g, varargin)
+            m = m@Measure(g, varargin{:});
+        end
+    end
+    methods (Access=protected)
+        function calculate(m)
+            g = m.getGraph();
+            A = g.getA();
+            N = length(A);
```
Comment (giovannivolpe) [inline braph/graph/measures/Triangles.m]: N = g.nodenumber();

File: braph/graph/measures/Triangles.m [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            if isa(g, 'GraphBU') || isa(g, 'GraphWU')
+                A3 = (A.^(1/3))^3;
+                temp = diag(A3)'/2;
```
Comment (giovannivolpe) [inline braph/graph/measures/Triangles.m]: change temp to clustering

File: braph/graph/measures/Triangles.m (hunk 1 above)
Comment (mitemijalkov) [reply braph/graph/measures/Triangles.m]: The change was made.

File: braph/graph/measures/Triangles.m (hunk 2 above)
Comment (mitemijalkov) [reply braph/graph/measures/Triangles.m]: This change was made.

**Pull request merged**